### PR TITLE
Initial commit of Flow Measuremnt Cluster (Issue #221).

### PR DESCRIPTION
**Pull request by JimBuzbee**
Tested in Google Home with a Flow Sensor device.

### CODE REVIEW

File: src/matter/cluster/FlowMeasurementCluster.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, OptionalAttribute, Cluster } from "./Cluster";
+import { MatterApplicationClusterSpecificationV1_0, TlvUInt16, TlvNullable, } from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for Flow Measurement.
```
Comment (Apollon77) [inline src/matter/cluster/FlowMeasurementCluster.ts:11]:
```suggestion
 * This cluster provides an interface to flow measurement functionality.
```

File: src/matter/cluster/FlowMeasurementCluster.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+       /** MeasuredValue represents the flow in m3 /h as follows: MeasuredValue = 10 x Flow */
+       measuredValue: Attribute(0, TlvNullable(TlvUInt16), { default: null }),
+       
+       /** Indicates the minimum value of MeasuredValue that can be measured. */
+       minMeasuredValue: Attribute(1, TlvNullable(TlvUInt16.bound({ min: 0 }))),
+       
+       /** Indicates the maximum value of MeasuredValue that can be measured. */
+       maxMeasuredValue: Attribute(2, TlvNullable(TlvUInt16.bound({ max: 65534 }))),
+
+       tolerance: OptionalAttribute(3, TlvUInt16.bound({ min: 0, max: 2048 }), {default: 0 }),
```
Comment (Apollon77) [inline src/matter/cluster/FlowMeasurementCluster.ts:31]:
```suggestion
       tolerance: OptionalAttribute(3, TlvUInt16.bound({ min: 0, max: 2048 }), { default: 0 }),
```

**Review by Apollon77 — CHANGES_REQUESTED** [review]
Two nit's :)

File: src/matter/cluster/FlowMeasurementCluster.ts [hunk 3]
```diff
@@ -19,13 +19,13 @@ export const FlowMeasurementCluster = Cluster({
 
     /** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.5.4 */
     attributes: {
-       /** MeasuredValue represents the flow in m3 /h as follows: MeasuredValue = 10 x Flow */
```
Comment (Apollon77) [inline src/matter/cluster/FlowMeasurementCluster.ts]: @JimBuzbee nnooooooooo :-)) in this case where the comment comes directly before a codeline and is describing this codeline the /** .. */ for is the correct one!!

Comment (JimBuzbee) [reply src/matter/cluster/FlowMeasurementCluster.ts]: :-) JSDoc - Add one more thing that I am a novice at! 

Correction pushed

**Comment by Apollon77** [comment]
Ok, if you accept the above two suggetsions then I think I can finalize the review

**Comment by JimBuzbee** [comment]
OK - Accepted

**Review by Apollon77 — APPROVED** [review]

**Pull request merged**